### PR TITLE
Specify what is meant by support

### PR DESCRIPTION
**Pull request by torben-hansen**
### Description of changes: 

Specify what is meant by "support".

### Testing:

No code was harmed in this PR.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by DominicDams — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -43,48 +47,34 @@ team to publish open source contributions and enhancements that are submitted to
...
+ARMv8 optimizations and Intel CPUs to test x86 and x86-64 optimizations.
 
-A portable C implementation of all algorithms is included and optimized assembly is
-included for some platforms. We test all changes on ARM and x86 platforms. We use
-[AWS Graviton processors](https://aws.amazon.com/ec2/graviton/) to test ARMv8 and
-Intel CPUs to test x86 and x86-64 compatibility. The
-[Intel Software Development Emulator](https://software.intel.com/content/www/us/en/develop/articles/intel-software-development-emulator.html)
-is used to test all modern x86 processors.
+The [Intel Software Development Emulator](https://software.intel.com/content/www/us/en/develop/articles/intel-software-development-emulator.html)
+is used to run tests all modern x86 processors.
```
Comment (dkostic) [inline README.md:73]:
```suggestion
is used to run tests on many different x86 processors.
```
we try to run the tests on all Intel models that came out in the last 15-20 years.

Comment (torben-hansen) [reply README.md]: Makes sense

File: README.md [hunk 2]
```diff
@@ -43,48 +47,34 @@ team to publish open source contributions and enhancements that are submitted to
...
-build libssl and AWS-LC’s tests you will need a C++ compiler. AWS-LC is tested with:
+### Compiler, OS, and CPU support
 
-* GCC 4.8.5 to 8.4
-* Clang 7 to 10
-* Visual Studio 2015
-
-For a complete list of tested compilers see
+AWS-LC correctness is tested on a variety of C/C++ compiler, OS, and CPU
+combinations. For a complete list of tested combinations see
```
Comment (darylmartin100) [inline README.md:60]: Can you please modify the ci/Readme to add Android as a platform?  Have we added other platforms since?

File: README.md [hunk 3]
```diff
@@ -11,6 +11,10 @@ applications. For performance critical algorithms, optimized assembly versions a
 included for x86 and ARM.
 
 ## Quickstart for Amazon Linux 2
+
+AWS-LC’s libcrypto is a C library and needs a C compiler. AWS-LC's libssl is a
```
Comment (darylmartin100) [inline README.md:15]: Nit:  Should call out libdecript here until we get rid of it?

Comment (torben-hansen) [reply README.md:15]: It's gone

File: tests/ci/README.md [hunk 4]
```diff
@@ -102,6 +112,7 @@ pull in any files from the seed folder and merge it into the growing corpus in E
 
 
 ### Cryptofuzz
+
 Each change is built and tested with [Cryptofuzz](https://github.com/guidovranken/cryptofuzz) for an hour. A seed corpus
 is included in tests/docker_images/cryptofuzz_data.zip. As new inputs are found they are saved in a shared corpus across
 runs in AWS EFS. Cryptofuzz is built with 3 modules:
```
Comment (samuel40791765) [inline tests/ci/README.md:118]: We also added ARM for cryptofuzz recently, so we can update this too :D https://github.com/awslabs/aws-lc/commit/8a7633d25ead8ed446a85dcb4ab6c2e8d2e4cdfc

Comment (torben-hansen) [reply tests/ci/README.md:118]: Ahh nice, missed that.

**Review by DominicDams — DISMISSED** [review]

**Review by samuel40791765 — DISMISSED** [review]

**Review by DominicDams — APPROVED** [review]

**Review by samuel40791765 — APPROVED** [review]

**Pull request merged**